xbmc/guilib/GUIColorButtonControl.cpp

```diff
@@ -35,7 +38,7 @@ CGUIColorButtonControl::CGUIColorButtonControl(int parentID,
   m_colorPosY = 0;
   m_imgColorMask->SetAspectRatio(CAspectRatio::AR_KEEP);
   m_imgColorDisabledMask->SetAspectRatio(CAspectRatio::AR_KEEP);
-  m_imgBoxColor = UTILS::COLOR::NONE;
+  m_imgBoxColor = ::GUIINFO::CGUIInfoColor(UTILS::COLOR::NONE);
```

Comment:
Can you drop the starting :: in all lines? Shouldn't be needed

Reply:
thanx, stupid find&replace error. fixed.